package.json

```diff
@@ -40,6 +44,7 @@
     "feedparser": "^2.2.1",
     "request": "^2.81.0",
     "request-promise": "^4.2.1",
+    "require-self-ref": "^2.0.1",
```

Comment:
Just realized you're using this module already but just now adding it to package.json. An automated test job would have caught this. We should get travis set up to run jobs automatically on push/pr, like the other repos.

Reply:
I'll make a separate ticket for this.